centrifuge-app/src/components/LoanList.tsx

```
@@ -55,9 +56,9 @@ const getLoanStatus = (loan: Loan | TinlakeLoan) => {
 
 export function LoanList({ loans }: Props) {
   const { pid: poolId } = useParams<{ pid: string }>()
-  const pool = usePool(poolId)
-  const isTinlakePool = poolId.startsWith('0x')
-  const basePath = useRouteMatch(['/pools', '/issuer'])?.path || ''
+  const pool = usePool(`${poolId}`)
```

Better to throw if `poolId` is `undefined` is such cases, to narrow the type of `poolId` to `string`, instead of `string | undefined`